items.rb

```diff
@@ -0,0 +1,36 @@
+require 'date'
+
+class Item
+  attr_accessor :id, :genre, :author, :source, :label, :publish_date, :archived
+
+  def initialize(id, genre, author, source, _label, publish_date, archived)
+    @id = id
+    @genre = genre
+    @author = author
+    @source = source
+    @label = labelrequire_relative '../label'
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def can_be_archived?
+    age_in_years = (Date.today - @publish_date).to_i / 365
+    age_in_years > 10
+  end
+
+  def genre(genre)
```

Comment:
Since you already add these `genre author source label` method on the `attr_accessor` you don't need to add set all methods again as it might create confusion.